docs/admin/audit-logs.md

```diff
@@ -0,0 +1,24 @@
+# Audit Logs
+
+Audit logs is an enterprise feature that allows **Admins** and **Auditors** to monitoring what is happening on their deployment.
+
+## Tracked Events
+
+The following events are tracked by this feature:
+
+- Create
+- Update
+- Delete
+
+For the following resources:
+
+- GitSSHKey
+- Template
+- TemplateVersion
+- Workspace
+- APIKey
+- User
+
+## Enabling this feature
+
+To enable this feature an Admin needs to purchase an enterprise license.
```

Comment:
And then what? It's automatically enabled?

Maybe like 
```suggestion
This feature is autoenabled for all enterprise deployments. An Admin can purchase a license [here](link)
```

Reply:
Yeah, we don't have a link yet :(

Reply:
Ah gotcha. Yeah whatever you think!